images/02-console/docker-init

```diff
@@ -11,10 +11,15 @@ done
 
 DOCKER_BIN=$(which docker) || DOCKER_BIN=/usr/bin/docker
 
-for i in /opt/bin /usr/local/bin; do
+for i in /opt/bin /usr/local/bin /var/lib/rancher/docker; do
+    if [ -x ${i}/dockerd ]; then
```

Comment:
Why do we need to copy these?  Can we not just add then to the PATH in the front?